core/README.md

```diff
@@ -10,7 +10,7 @@ This example contains the code of a simple amplification plugin. Please note tha
 
 ```Rust
 // Import everything we need.
-use lv2_core::prelude::*;
+use lv2::core::prelude::*;
```

Comment:
The use of `lv2_core` instead of `lv2::core` is intentional here. Our line of thought was that one would use the `lv2` crate only for prototyping or to get started and then use the `lv2_` crates to create the proper plugins. Please use `lv2_core` instead.
```suggestion
use lv2_core::prelude::*;
```

Reply:
lol, that's not me

Reply:
solved on my branch, I also fixed the README at root.